src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -116,6 +105,7 @@ private static string GetItemKind(DeclaredSymbolInfo declaredSymbolInfo)
                 case DeclaredSymbolInfoKind.Interface:
                     return NavigateToItemKind.Interface;
                 case DeclaredSymbolInfoKind.Constructor:
+                case DeclaredSymbolInfoKind.ExtensionMethod:
```

Comment:
What was the consequence of not having this?  Do we ever have things "declared" as Extension methods?